### ISSUE TITLE
Bad recognize

### ISSUE THREAD

**Issue by lfdominguez** [deniyuda348/libfprint#4]
A great job +10 for you
I have an HP ENVY 17 PC and wen i use the fprint i need pass my finger much time, almost that not recongnize me.... 
Then, in fprint-demo, when scan my finger, almost the image that show is distorcioned, very very large, depending of the time of scroll my finger overall sensor.

**Comment by nathanalderson** [comment]
Agreed!  Thanks for the work you've done.  I got it compiled and installed on a nearly fresh Mint-17 system, but it was very unreliable.  After registering my finger (no problems), it would only successfully match a fingerprint scan about 1 in 5 times.  I had to disable it. :(

**Comment by lfdominguez** [comment]
Me too, the scanned image is like that don't have a code to get a propel image trhought the time, if i get much time passing my finger throw the scanner the image is very big and distorded.
Thanks for the implement specific to the 138a:0050 device.... But in a future, when discuss my diploma thesis in Informatic Science i got the code to see how fix the scanner...-----------------------------------------------------------On Miércoles, 3 de junio de 2015 06:43:11 AM Nathan Alderson wrote:


Agreed! Thanks for the work you've done. I got it compiled and installed on a nearly fresh Mint-17 system, but it was very unreliable. After registering my finger (no problems), it would only successfully match a fingerprint scan about 1 in 5 times. I had to disable it. :( 
—Reply to this email directly or view it on GitHub[1].[2]




--------------------
*Luis Felipe Domínguez Vega*
lfdominguez@estudiantes.uci.cu[3] 

--------
[1] https://github.com/payden/libfprint/issues/4#issuecomment-108420868
[2] https://github.com/notifications/beacon/AFWDIyoXR6DupYActXqXfNBI5_17Z8Wdks5oPvvvgaJpZM4CfzZE.gif
[3] mailto:lfdominguez@estudiantes.uci.cu

**Comment by angelodelia** [comment]
I agree, once installed.... it doesn't work properly.

But... how did you managed to remove it???

I got the terrible surprise that the driver is called each and every time you login or sudo. If you uninstall fprintd using the package manager, the reader is no longer activated, but, surprise, surprise, the driver is "called" in such way, I see a lot of debug lines in terminal. 

And... if you try to "destroy" manually the libfprint.so* .... you are kicked out of your user, because when login do not find them, it will NOT fallback to asking password.

Any idea?
